nodeprobe/nodeprobe.go

```diff
@@ -136,3 +136,27 @@ func (p *Prober) AddNode(name string, node Node) {
 
 	p.nodes[name] = node
 }
+
+func (p *Prober) CheckBeaconNodeHealth(ctx context.Context) error {
+	err := p.nodes["consensus client"].Healthy(ctx)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
```suggestion
	return p.nodes["consensus client"].Healthy(ctx)
```

or let's wrap the error